_common/README

```diff
@@ -0,0 +1,38 @@
+<!-- README Template for ory/examples. -->
+<!-- Use the following template for all READMEs in ory/examples. -->
+
+# Title 
+
+<!--Add one sentence describing the project, usecase and key services used. -->
+<!--Add a link to post on the Ory Blog as well as other platforms (dev.to, ...). -->
+
+## Overview
+
+<!-- Optional: Add an diagram if using multiple services or a more detailed explanation than in description or links to other relevant resources. -->
+
+### Prerequisites
+
+<!--Add a list of software needed to run the example, ideally with links to the installation page or installation (e.g. brew) commands. -->
+
+## Develop
+
+<!-- Add a numbered list of commands needed to run the example locally or with Ory Cloud playground. -->
+
+### Environmental Variables
+
+<!-- Optional: If needed add instructions on how to use environmental variables. -->
+
+### Run tests
+
+<!-- Add a numbered list of commands needed to run tests locally. -->
```

Comment:
Why a numbered list? There are often several tests that one could run, e.g. unit tests, end-to-end tests, documentation tests, all of the above tests, etc. Those are typically not run in any particular order.